Makefile-test.mk

```diff
@@ -137,6 +140,14 @@ run-test-e2e-singlecluster-%: FORCE
 		./hack/e2e-test.sh
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/$@/e2e.json > $(ARTIFACTS)/$@/e2e-top.yaml
 
+run-test-e2e-ocp:
```

Comment:
Could you look into adding a new Makefile for ocp?

ie 

Makefile-ocp.mk?

I worry that we could lose these changes during a rebase pretty easily.